low-level/files/opt/seagate/sspl/bin/generate_resource_health_view/resource_health_view

```diff
@@ -847,6 +847,7 @@ class SSPLHealthView(object):
         if response.status_code != singleton_realstorencl.ws.HTTP_OK:
             if (response.status_code == singleton_realstorencl.ws.HTTP_TIMEOUT or \
                 response.status_code == singleton_realstorencl.ws.HTTP_CONN_REFUSED or \
+                response.status_code == singleton_realstorencl.ws.HTTP_FORBIDDEN or \
```

Comment:
how forbidden error would vanish even if you try another controller in enclosure, same user account credentials will apply there also

Reply:
Yes, I agree,
@indrajitzagade saw (111) https://jts.seagate.com/browse/EOS-13209 (Error pasted in descriptions) error. I have tested the health view script in all possible ways and I got one scenario that when IPs are configured correctly and the password is not then the health view script is failing with the given error (HTTP_FORBIDDEN). after making the above changes this error got resolved.

Reply:
@satish-seagate 
_I got one scenario that when IPs are configured correctly and the password is not then the health view script is failing with the given error (HTTP_FORBIDDEN). after making the above changes this error got resolved._

Please mention this code change helped in your testing?

Reply:
whats the logic ? It seems on one controller, one password was configured, while controller was supporting some other password? This different password per controller i dont think is possible.

Make sure you were using inband controller Ips and not out of band IPs, with some controller IPs from different enclosures !?

Reply:
@satish-seagate The JIRA was filed for the case where the ws request was failed with "Realstor Storage_enclosure:: http request for login failed with err 111". Looks like already we have a retry in place for it to try with other controller. Not sure still why this retry did not help where the issue observed that time.

Reply:
can we conclude this soon?

Reply:
@satish-seagate  @cdeshmukh 
The fix being added through this PR is missing from main branch while it is present in the cortx-1.0 branch, looks like a merge from cortx-1.0 branch to main branch was missed somehow. So this PR is needed I think.
But the original issue reported through EOS-13209 could be different, if not reproducing now we can merge the current PR and conclude the issue.

Reply:
@cdeshmukh @indrajitzagade 
I debugged the code for **cortx-1.0** branch and I saw that after the https://github.com/Seagate/cortx-monitor/pull/242 @ArchanaLimaye change we don't need this retry mechanism login code in health view. https://github.com/Seagate/cortx-monitor/pull/242 will take care of that internally in the enclosure code.
We can remove this code for cortx-1.0 because https://github.com/Seagate/cortx-monitor/pull/242 is already merged in cortx-1.0 but we can't remove this code from the main branch because I am not seeing https://github.com/Seagate/cortx-monitor/pull/242 change in the main branch yet. To removing this code from main we need to merge https://github.com/Seagate/cortx-monitor/pull/242 in main first.

Reply:
> 
> 
> @cdeshmukh @indrajitzagade
> I debugged the code for **cortx-1.0** branch and I saw that after the #242 @ArchanaLimaye change we don't need this retry mechanism login code in health view. #242 will take care of that internally in the enclosure code.
> We can remove this code for cortx-1.0 because #242 is already merged in cortx-1.0 but we can't remove this code from the main branch because I am not seeing #242 change in the main branch yet. To removing this code from main we need to merge #242 in main first.

@satish-seagate I think fix in #242 will be needed in main branch too. Then we can remove this retry code on top of that port.

Reply:
cortx-1.0 branch is hard to touch now, until its a must fix. And anyways with #242 fix, this PR may not even apply to cortx-1.0 rt, so we should be fine there. In main branch though, lets pull #242 only then, and in same PR remove this retry mechanism

Reply:
@ArchanaLimaye @indrajitzagade @cdeshmukh 
#242 is already applied in cortx-1.0. To remove the retry enclosure login mechanism from the health view script. @ArchanaLimaye we need this PR in main too.

Reply:
@cdeshmukh @indrajitzagade @ArchanaLimaye 
I have applied #242 changes in main and removed the retry login mechanism from the health view script.
Please review.